src/sentry/static/sentry/app/types/index.tsx

```diff
@@ -748,7 +748,14 @@ export type Integration = {
   provider: BaseIntegrationProvider & {aspects: any};
   configOrganization: Field[];
   //TODO(ts): This includes the initial data that is passed into the integration's configuration form
-  configData: object;
+  configData: object & {
+    //installationType is only for Slack migration and can be removed after migrations are done
+    installationType?:
```

Comment:
damn, I kinda dislike this existing here, but 🤷‍♂️

Reply:
@EvanPurkhiser I mean, what's the alternative? But it's temporary so it's not that bad IMO.